### DISCUSSION TITLE
带有初始值的层级选择器懒加载时reject的行为

### DISCUSSION THREAD

**Discussion by KawaiiZapic** [element-plus/element-plus#6578] [General]
我给cascader写了个reject: [https://github.com/KawaiiZapic/element-plus/tree/feat/el-cascader-lazyload](https://github.com/KawaiiZapic/element-plus/tree/feat/el-cascader-lazyload) 
但是我还没有想明白如果带有初始值的选择器被reject了应该发生什么, 希望能从大家这里得到一些想法.
`<el-cascader :model-value="[1, 2, 4]" />`

**Comment by Dsaquel** [comment]
https://github.com/element-plus/element-plus/pull/22283
